elasticsearch/Dockerfile

```diff
@@ -1,3 +0,0 @@
-FROM docker.elastic.co/elasticsearch/elasticsearch:5.6.0
```

Comment:
What's the purpose for deleting stuff behind `elasticsearch/`?

Reply:
Not needed anymore.
It was used in `docker-compose` only, but now it is possible to just use an official docker image.